app/src/main/res/layout/view_suggested_edits_disabled_states.xml

```diff
@@ -11,65 +11,40 @@
     android:orientation="vertical">
 
     <ImageView
-        android:id="@+id/image"
+        android:id="@+id/imageView"
         android:layout_width="match_parent"
-        android:layout_height="92dp"
-        android:layout_marginBottom="20dp"
+        android:layout_height="wrap_content"
         android:contentDescription="@null"
-        app:srcCompat="@drawable/ic_suggested_edits_disabled" />
+        android:layout_marginBottom="16dp" />
 
     <TextView
         android:id="@+id/messageTitleView"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:lineSpacingExtra="8sp"
-        android:layout_marginBottom="16dp"
-        android:textColor="?attr/material_theme_primary_color"
+        android:textSize="20sp"
+        android:fontFamily="sans-serif"
         android:textStyle="bold"
-        android:textSize="20sp" />
+        android:textColor="?attr/material_theme_primary_color"
+        android:lineSpacingExtra="8sp"
+        android:layout_marginBottom="12dp"/>
 
     <TextView
         android:id="@+id/messageTextView"
         android:layout_width="match_parent"
         android:layout_height="wrap_content"
-        android:lineSpacingExtra="8sp"
+        android:textSize="16sp"
+        android:fontFamily="sans-serif"
         android:textColor="?attr/material_theme_secondary_color"
-        android:textSize="16sp" />
+        android:lineSpacingExtra="8sp"
+        android:layout_marginBottom="12dp"/>
 
-    <LinearLayout
-        android:id="@+id/learnMoreContainer"
+    <com.google.android.material.button.MaterialButton
```

Comment:
This is a good find.  If you've got some spare cycles, perhaps consider a follow-up PR that updates our other usages of these buttons (e.g. in `view_suggested_edits_task_item.xml`) to use `MaterialButton`.